inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/core_threading_tests.cpp

```diff
@@ -5,19 +5,20 @@
 #include <behavior/core_threading_tests.hpp>
 
 namespace {
-
 Params params[] = {
     std::tuple<Device, Config>{ CommonTestUtils::DEVICE_GNA, {{ CONFIG_KEY(PERF_COUNT), CONFIG_VALUE(YES) }}},
     std::tuple<Device, Config>{ CommonTestUtils::DEVICE_HETERO, {{ "TARGET_FALLBACK", CommonTestUtils::DEVICE_GNA }}},
     std::tuple<Device, Config>{ CommonTestUtils::DEVICE_MULTI, {{ MULTI_CONFIG_KEY(DEVICE_PRIORITIES), CommonTestUtils::DEVICE_GNA }}},
 };
-
+// TODO: Consider to append params[1] after issue *-45658 resolved
+std::vector< std::tuple<Device, Config> > paramsWithIterations{ params[0], params[2] };
 }  // namespace
 
 INSTANTIATE_TEST_CASE_P(GNA, CoreThreadingTests, testing::ValuesIn(params), CoreThreadingTests::getTestCaseName);
 
-INSTANTIATE_TEST_CASE_P(DISABLED_GNA, CoreThreadingTestsWithIterations,
-    testing::Combine(testing::ValuesIn(params),
-                     testing::Values(2),
-                     testing::Values(2)),
+INSTANTIATE_TEST_CASE_P(GNA, CoreThreadingTestsWithIterations,
```

Comment:
I've disabled the tests back https://github.com/openvinotoolkit/openvino/pull/3661/files#diff-1a9c8f22719ab45faaff9213da60d2315f42b589765da498cdea7c94ab78e5a3R19. They fails from time to time on CI.

Reply:
@ilya-lavrenov Sure.
Could you point to some failing build(s) ? It would be easier to repro.

Reply:
https://openvino-ci.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fie-tests-linux-ubuntu18-gna/detail/ie-tests-linux-ubuntu18-gna/23570/pipeline